QualityOntology.ttl

```diff
@@ -736,7 +736,7 @@ cco:SpatialOrientation rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/Spherical
 cco:Spherical rdf:type owl:Class ;
               rdfs:subClassOf cco:ThreeDimensionalShape ;
-              cco:definition "A Three Dimensional Shape inhering in a bearer in virtue of it having round cross sections."@en ;
+              cco:definition "A Three Dimensional Shape inhering in a bearer in virtue of it having only round cross sections."@en ;
```

Comment:
@cameronmore It appears @gregfowlerphd offered a further candidate paraphrase on the issue. Please see what you think.

Reply:
adjusted